content/forms.py

```diff
@@ -4,8 +4,8 @@
 
 
 class OurTeamForm(forms.ModelForm):
```

Comment:
This was part of jiwon's code. I think its for the admin page entry of team names.